synapse/config/registration.py

```diff
@@ -15,6 +15,8 @@
 
 from distutils.util import strtobool
 
+from synapse.config._base import ConfigError
+from synapse.types import RoomAlias
 from synapse.util.stringutils import random_string_with_symbols
 
 from ._base import Config
```

Comment:
it'd be nice to merge this into line 18 while you're at it (absolute imports are preferred over relative ones)